app/controllers/phases_controller.rb

```diff
@@ -65,13 +61,18 @@ def set_submission_counts
   end
 
   def set_submission_statuses
-    @not_started = @submissions.left_joins(evaluator_submission_assignments: :evaluation).
-      where(evaluations: { id: nil }).distinct
-    @in_progress = @submissions.joins(evaluator_submission_assignments: :evaluation).
-      where(evaluations: { completed_at: nil }).distinct
-    @completed = @submissions.joins(evaluator_submission_assignments: :evaluation).
-      where.not(evaluations: { completed_at: nil }).
-      where.not(id: @in_progress.select(:id)).distinct
+    eligible_submissions = @submissions.eligible_for_evaluation
+
+    @not_started = eligible_submissions.where(evaluation_status: :not_started)
+    @in_progress = eligible_submissions.where(evaluation_status: :in_progress)
+    @completed = eligible_submissions.where(evaluation_status: :completed)
```

Comment:
because this is an `enum` field, you get some query methods automatically that match the statuses
```suggestion
    @not_started = eligible_submissions.not_started
    @in_progress = eligible_submissions.in_progress
    @completed = eligible_submissions.completed
```